Create da.json

hrassel opened the pull request:
A danish translation of a very promising plugin

## Description

Just a danish translation for the project, noting else, nothing more. 

## Related Issue

No issues.

alexpfau commented:
Hi @hrassel,

Thank you for your translation contribution! I really appreciate your help in making Calendar Card Pro more accessible to users around the world.

Before I can merge your PR, there's a small update needed. I recently released v1.1.0 which fixed some bugs and enhanced the multi-day event display. As part of this work, **two new language keys** were added that need to be included in all language files:

- `endsToday`: For events that end on the current day (e.g., "ends today" in English)
- `endsTomorrow`: For events ending the next day (e.g., "ends tomorrow" in English)

Could you please update your translation PR to include these two new keys? You can see examples in the [CONTRIBUTING.md](https://github.com/alexpfau/calendar-card-pro/blob/main/CONTRIBUTING.md#adding-new-translations) file.

I apologize for the delay in processing language PRs - I had to prioritize fixing some critical bugs, but I'm now focusing on integrating new languages. Once you've added these two strings, I'll prioritize reviewing and merging your contribution.

Thanks again for helping make Calendar Card Pro better!

hrassel commented:
> Hi @hrassel,
> 
> Thank you for your translation contribution! I really appreciate your help in making Calendar Card Pro more accessible to users around the world.
> 
> Before I can merge your PR, there's a small update needed. I recently released v1.1.0 which fixed some bugs and enhanced the multi-day event display. As part of this work, **two new language keys** were added that need to be included in all language files:
> 
> * `endsToday`: For events that end on the current day (e.g., "ends today" in English)
> * `endsTomorrow`: For events ending the next day (e.g., "ends tomorrow" in English)
> 
> Could you please update your translation PR to include these two new keys? You can see examples in the [CONTRIBUTING.md](https://github.com/alexpfau/calendar-card-pro/blob/main/CONTRIBUTING.md#adding-new-translations) file.
> 
> I apologize for the delay in processing language PRs - I had to prioritize fixing some critical bugs, but I'm now focusing on integrating new languages. Once you've added these two strings, I'll prioritize reviewing and merging your contribution.
> 
> Thanks again for helping make Calendar Card Pro better!

Hi @alexpfau 

I have updated the translation as mentioned, and I have corrected a couple things in the previous wording that in danish could be cause of confusion. 
So if you want, you can use that. 
I hope it is useful, and thank you for the other fine work you are doing, it really is a nice addon you are working on here.
